identity-verification/module_3/3.2 Fun with Rekognition DetectFaces and SearchFacesByImage.ipynb

```diff
@@ -0,0 +1,430 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 3.2 Fun with Rekognition DetectFaces and SearchFacesByImage\n",
+    "----\n",
+    "This is fun but optional module. Here we are going to take a look at extracting and indexing multiple faces, from an image. Once we have the collection of images we'll compare all faces in an image with Amazon Rekognition against your collection.\n",
+    "\n",
+    "## Steps \n",
+    "\n",
+    "1. Load packages \n",
+    "2. View existing collections \n",
+    "3. Create a new collection \n",
+    "4. Download some images and unzip them into a directory \n",
+    "5. Index the faces found into the collection \n",
+    "6. Find and present faces found in an image vs. the faces stored in your collection \n",
+    "7. Clean up!! \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following notebook will guide you through on how to compare all faces detected in an image against your Amazon Rekognition Face Collection. \n",
+    "\n",
+    "## Step 1. Load Libraries "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import boto3, json, os, io\n",
+    "from PIL import Image, ImageDraw, ExifTags, ImageColor\n",
+    "from IPython.display import Image as IImage\n",
```

Comment:
These imports aren't used anywhere. Can they be removed?

Reply:
removed.